core/controllers/admin.py

```diff
@@ -489,19 +489,19 @@ def _load_dummy_new_structures_data(self):
             self._reload_exploration('6')
             self._reload_exploration('25')
             self._reload_exploration('13')
-            exp_services.update_exploration(
+            exp_services.compute_models_for_updating_exploration(
```

Comment:
I don't understand what's happening here ... "compute models" doesn't sound like "update the datastore" but it sounds like that's what is happening here. So I don't think this works.

I think you need an update_exploration() method and a compute_models_for_updating_exploration() method, where the former calls the latter internally. Any disadvantages of doing that?